deploy.php

```diff
@@ -52,6 +66,8 @@
 
 after('deploy:vendors', 'frontend:build');
 after('frontend:build', 'app');
+after('app', 'migrations');
+after('migrations', 'fixtures');
```

Comment:
```suggestion
after('app', 'fixtures');
```

I think it is better that way

Reply:
Fixed!